optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/common/accessor/lambda/ShortPropertySetterAccessor.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.core.impl.domain.common.accessor.lambda;
+
+import java.lang.invoke.CallSite;
+import java.lang.invoke.MethodHandles;
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Method;
+
+import org.optaplanner.core.impl.domain.common.accessor.lambda.consumer.ObjectShortConsumer;
+
+final class ShortPropertySetterAccessor extends PropertySetterAccessor<ObjectShortConsumer> {
+
+    ShortPropertySetterAccessor(final Method setterMethod,
```

Comment:
we don't use the friendly modifier normally. Let's make this constructor public. It's in the "impl" package, so it's not part of the API.

Reply:
the same argument as above